backend/cmd/api/loaders.go

```diff
@@ -37,6 +37,9 @@ func getLoadersForRoles(db *sql.DB, queries *sqlc.Queries, collectionLoader *loa
 	rq := queries.RoleQueries(roles)
 
 	ls := &common.FilteredLoaders{
+		GetKey: func() string {
```

Comment:
I can't quite see the reason as to why this is a function and not just a string.

Reply:
For interface implementation, but probably not needed, so can just switch to a string.